core/src/main/resources/defaults/vm-logs.json

```diff
@@ -310,5 +310,9 @@
   {
     "path": "/var/log/jumpgate/out.log",
     "label": "jumpgate_agent_out"
+  },
+  {
+    "path": "/var/log/mount-instance-storage.log",
```

Comment:
only DL and DH related? if freeipa as well, aadd this to its vm-logs.json file as well

Reply:
it's only DL and DH related, similarly to mount-disks.log